### ISSUE TITLE
Does not support IAM profiles

### ISSUE THREAD

**Issue by jamiesonbecker** [smore-inc/tinys3#47]
Maybe something that's planned.

**Comment by duboff** [comment]
Works for me. What problem did you have?

**Comment by jamiesonbecker** [comment]
Tinys3 didn't seem to even attempt or successfully find the IAM instance profile (an IAM role+policy with a rotating set of credentials that is passed in through instance metadata without a preset ID or key), so we switched back to boto. If you're saying that I'm wrong, that's awesome and we'll try again!

**Comment by duboff** [comment]
No, I think my use case was just a lot simpler, so wouldn't know. Just saw your issue after having a problem in that area that I then solved by changing permissions on the AWS side.

**Comment by jamiesonbecker** [comment]
cool, no worries :) Thinking that there needs to be a less complex wrapper around boto, since TinyS3 was pretty nice and fun to work with.
